app/helpers/provider/admin/dashboards_helper.rb

```diff
@@ -19,17 +19,18 @@ def sign_class(widget)
     widget.percentual_change > 0 ? 'u-plus' : 'u-minus'
   end
 
-  def dashboard_collection_link(singular_name, collection, path, plural = nil, icon_name = nil)
-    link_to path, class: 'DashboardNavigation-link' do
+  def dashboard_collection_link(singular_name, collection, path, icon_name: nil, plural: nil, notice: false)
+    className = 'DashboardNavigation-link' + " u-notice" if notice
```

Comment:
you can do put them in the same string, no? plus it's better to use just single quotes in this case

Reply:
```ruby
 def dashboard_collection_link(singular_name, collection, path, icon_name: nil, plural: nil, classNames: [])
   # etc ...
end
```

You can even use `Array` `(['DashboardNavigation-link'] + Array(class_names)).compact` and rails will take care of joining them automatically

Reply:
@hallelujah That would return an Array instead of a String, which is ok if we are gonna convert it to string later, but then if we are gonna do it by `.join(' ')`, we probably don't even need the `compact`, no? because all it does is removing the nil values, which I don't think we would need it 🤔 Maybe we can even simplify it to`"DashboardNavigation-link #{class_names.join(' ')}"`

Reply:
Rails takes care of it in `link_to`